### PR TITLE
[Feature] User Login

### PR DESCRIPTION
**Pull request by vinibr3**
Create user login and navbar.

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  def destroy
+    sign_out
+    flash[:success] = t("activerecord.session.form.flash.exit")
+    redirect_to new_login_path
+  end
+
+  private
+
+    def session_params
```
Comment (kaiomagalhaes) [inline app/controllers/sessions_controller.rb]: please name it `valid_params` this way you don't need to have a params name for each controller

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -93,4 +93,5 @@
   # as the one that triggered the failure.
   Kernel.srand config.seed
 =end
+  # config.include Support::Features::SessionHelpers, type: :feature
```
Comment (kaiomagalhaes) [inline spec/spec_helper.rb]: why is this commented code here?

File: spec/support/feature.rb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+RSpec.configure do |config|
```
Comment (kaiomagalhaes) [inline spec/support/feature.rb:1]: shouldn't the filename be `features`?

**Review by kaiomagalhaes — CHANGES_REQUESTED** [review]

**Review by kaiomagalhaes — DISMISSED** [review]

**Review by RodrigoNM — DISMISSED** [review]

File: app/views/sessions/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+      <%= f.email_field :email, {placeholder: t('activerecord.session.form.email.placeholder'), class: 'form-control'} %>
+    </div>
+  <div class="form-group">
+      <%= f.label :password, t('activerecord.session.form.password.label') %>
+      <%= f.password_field :password, { placeholder: t('activerecord.session.form.password.placeholder'), class: 'form-control'} %>
+  </div>
+  <div class="form-group">
+      <%= f.submit t('activerecord.session.form.submit'), class: 'btn btn-info' %>
+  </div>
+  <%end%>
```
Comment (matcheus) [inline app/views/sessions/new.html.erb]: This tag is with wrong identation and should have spaces between "<%" and "end", like this "<% end %>"

File: app/views/sessions/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  </div>
+</div>
```
Comment (matcheus) [inline app/views/sessions/new.html.erb]: This file has errors.

File: app/views/users/show.html.erb [hunk 6]
```diff
@@ -0,0 +1 @@
+<h1> Hello </h1>
```
Comment (matcheus) [inline app/views/users/show.html.erb]: This file has a error too.

File: spec/features/user_sign_in_spec.rb [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+    sign_in_with(Faker::Internet.email, password)
+    expect(page).to have_content(t("activerecord.session.form.flash.login_error"))
+  end
+
+  scenario "with invalid password" do
+    sign_in_with(user.email, Faker::Internet.password)
+    expect(page).to have_content(t("activerecord.session.form.flash.login_error"))
+  end
+
+  it "when user log out" do
```
Comment (matcheus) [inline spec/features/user_sign_in_spec.rb]: If you are doing acceptance tests with Capybara you should use "scenario" here too, instead of "it"

**Review by matcheus — CHANGES_REQUESTED** [review]

File: app/views/layouts/_navbar.html.erb [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+          <span> <%= t('buy_here.wellcome') %></span>
+          <%= current_user_email_helper %>
+        </a>
+        <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+          <%=  exit_helper %>
+        </div>
+      </li>
+    </ul>
+  </div>
+</nav>
```
Comment (matcheus) [inline app/views/layouts/_navbar.html.erb]: This file have errors too.

**Review by matcheus — CHANGES_REQUESTED** [review]

File: app/views/sessions/new.html.erb [hunk 9]
```diff
@@ -0,0 +1,18 @@
+<div class="row">
+  <div class="col-md-6 offset-md-3">
```
Comment (matcheus) [inline app/views/sessions/new.html.erb:2]: Identation of this file is wrong

**Review by matcheus — APPROVED** [review]

**Review by RodrigoNM — APPROVED** [review]

**Pull request merged**